### PR TITLE
fix: streak reset time calculation

### PR DESCRIPTION
**Pull request by sshanzel**
Root cause: the column was `timestamptz`, so we should not use the debezium time function. Returned value was string.

Still added some logging for debugging purposes. This should only log when the user streak was greater than 2 and turned to 0.

### CODE REVIEW

File: __tests__/workers/cdc/primary.ts [hunk 1]
```diff
@@ -3259,8 +3259,8 @@ describe('user streak change', () => {
     currentStreak: 2,
     totalStreak: 3,
     maxStreak: 4,
-    lastViewAt: new Date().getTime(),
-    updatedAt: new Date().getTime(),
+    lastViewAt: new Date().toISOString() as never,
+    updatedAt: new Date().toISOString() as never,
```
Comment (rebelchris) [inline __tests__/workers/cdc/primary.ts:3263]: Shouldn't it cast to `string` in this case?
Or TS doesn't like it?

File: src/common/users.ts [hunk 2]
```diff
@@ -9,16 +9,13 @@ import {
...
-  getTimezonedEndOfISOWeek,
-  debeziumTimeToDate,
-} from './utils';
+import { getTimezonedStartOfISOWeek, getTimezonedEndOfISOWeek } from './utils';
 import { GraphQLResolveInfo } from 'graphql';
 import { utcToZonedTime } from 'date-fns-tz';
 import { sendAnalyticsEvent } from '../integrations/analytics';
 import { DayOfWeek, DEFAULT_WEEK_START } from './date';
 import { ChangeObject, ContentLanguage } from '../types';
+import { logger } from '../logger';
```
Comment (rebelchris) [inline src/common/users.ts:18]: Where'd this guy come from?

**Review by rebelchris — APPROVED** [review]
Looks alright overall :)

File: __tests__/workers/cdc/primary.ts (hunk 1 above)
Comment (sshanzel) [reply __tests__/workers/cdc/primary.ts:3263]: TS doesn't like it.

File: src/common/users.ts (hunk 2 above)
Comment (sshanzel) [reply src/common/users.ts:18]: Oh, the logging was not pushed.

**Review by capJavert — APPROVED** [review]
Looks ok changes wise, trust you debugged it from feature perspective, since you have most knowledge of the edge cases.

**Pull request merged**